src/main/kotlin/day19/data_source/UserDataSource.kt

```
@@ -0,0 +1,5 @@
+package org.example.day19.data_source
+
+interface UserDataSource {
+
+}
```

detekt (1.23.7)

[warning] The class or object UserDataSource is empty.

(detekt.empty-blocks.EmptyClassBlock)